examples/lines_bars_and_markers/markevery_prop_cycle.py

```diff
@@ -41,13 +41,8 @@
           '#17becf',
           '#1a55FF']
 
-# Create two different cyclers to use with axes.prop_cycle
-markevery_cycler = cycler(markevery=cases)
-color_cycler = cycler('color', colors)
-
-# Configure rcParams axes.prop_cycle with custom cycler
-custom_cycler = color_cycler + markevery_cycler
-mpl.rcParams['axes.prop_cycle'] = custom_cycler
+# Configure rcParams axes.prop_cycle to simultaneously cycle cases and colors.
+mpl.rcParams['axes.prop_cycle'] = cycler(markevery=cases, color=colors)
```

Comment:
similar comment here as above. This isn't wrong, and it certainly should be advertised, but it is a convenience method and as such, it is limited.

Reply:
This example is a bit weird anyway. It seems to be written to prove that a certain bug/feature does now work. I'm not clear how useful this is.

Anyway, the main topic here is that markevery can be used. It's not about different ways of composing cyclers. Therefore, I chose the 'natural' form for the composition-usecase, which is multiple keyworsd here.